lib/timers.js

```diff
@@ -190,7 +192,8 @@ function insert(item, unrefed) {
     item[async_id_symbol] = ++async_uid_fields[kAsyncUidCntr];
     item[trigger_id_symbol] = initTriggerId();
     if (async_hook_fields[kInit] > 0)
-      emitInit(item[async_id_symbol], 'Timeout', item[trigger_id_symbol], item);
+      emitInit(item[async_id_symbol], asyncTypeTO, item[trigger_id_symbol],
```

Comment:
These changes are still confusing, imho. In any case, if you want to keep them, name them `kTimeoutType` or similar since they are constants. :)